src/site/markdown/syntax_highlight/mod.rs

```diff
@@ -24,9 +25,18 @@ fn find_syntax<'a>(ps: &'a SyntaxSet, language: &'a str) -> Result<&'a SyntaxRef
         return Ok(syntax_name);
     }
 
-    Err(OrandaError::Other(
-        "Please add the language to your code snippets".to_owned(),
-    ))
+    // this syntax will always be found as it's part of the default set
+    // https://github.com/sublimehq/Packages
+    let plain_text = ps
+        .syntaxes()
+        .iter()
+        .find(|syntax| syntax.name == "Plain Text")
+        .unwrap();
+
+    let msg = format!("Found syntax highlight language annotation `{language}` which is not currently supported. The annotated block will be shown as plaintext. Please file an issue https://github.com/axodotdev/oranda/issues/new to let us know you'd like to see it supported.");
```

Comment:
Maybe only show this if the user passed a language? If no language is passed then we are supposed to show it as plain text